webservices/server-integration/src/main/java/org/jboss/as/webservices/util/ModuleClassLoaderProvider.java

```diff
@@ -31,7 +31,7 @@ public ClassLoader getWebServiceSubsystemClassLoader() {
     public ClassLoader getServerIntegrationClassLoader() {
         if (integrationClassLoader == null || integrationClassLoader.get() == null) {
             try {
-                Module module = Module.getBootModuleLoader().loadModule(ASIL);
+                Module module = Module.getBootModuleLoader().loadModule(ModuleIdentifierUtil.canonicalModuleIdentifier(ASIL));
```

Comment:
It is not necessary to use ModuleIdentifierUtil helper class here. The string being used is standard module name.

Reply:
+1.

Generally we should reserve use of ModuleIdentifierUtil.canonicalModuleIdentifier to making sure external inputs are valid, where users might be passing "org.foo:main" etc.

Reply:
Fixed.